apps/opik-frontend/src/router.tsx

```diff
@@ -91,7 +90,12 @@ const workspaceRoute = createRoute({
 const quickstartRoute = createRoute({
   path: "/$workspaceName/quickstart",
   getParentRoute: () => workspaceGuardPartialLayoutRoute,
-  component: QuickstartPage,
+  component: () => (
+    <Navigate
+      to="/$workspaceName/home"
+      params={{ workspaceName: useAppStore.getState().activeWorkspaceName }}
+    />
+  ),
 });
```

Comment:
A nice catch, I'm not sure if it is needed tho. So maybe it's best to ask the product whether we should keep it. But anyway, we can remove it later